symbolic-examples/stats/UInt.DivMod.32.Auto

```diff
@@ -1,4 +1,4 @@
 UInt.DivMod.32.Auto
-Number of constraints: 999
-Number of variables: 1347
-Number of range lookups: 766
\ No newline at end of file
+Number of constraints: 96
```

Comment:
Does this only affect this benchmark because the others don't use division?

Reply:
Seems like it, yes
Benchmarks are run in CI so code checks wouldn't pass if anything else changed

Reply:
We have some benchmarks commented out though because they used too much RAM to compile, maybe some of them started working now